src/modules/ticket/actions.ts

```diff
@@ -0,0 +1,18 @@
+import {ThunkAction} from 'redux-thunk';
+import {RootAction, RootState} from '@store/rootReducer';
+import {buyTicketCall} from 'src/api/api';
+
+export const buy = (
+  flightId: string,
+): ThunkAction<void, RootState, unknown, RootAction> => (dispatch) => {
+  dispatch({type: 'BUY_TICKET_REQUEST', payload: flightId});
+
+  buyTicketCall(flightId)
+    .then((response) => {
+      dispatch({
+        type: 'BUY_TICKET_SUCCESS',
```

Comment:
Line:8, Line:13, Should be fine if an `ActionConstant` is created.
 > Constant variables on `Line:5` to `Line:7` should be moved to a separate file inside `modules/ticket`. It an be called something like `ActionConstant.ts`
  https://github.com/adescode/CRTASK/pull/1#discussion_r865464051